apps/clock_info/lib.js

```diff
@@ -332,6 +332,10 @@ exports.addInteractive = function(menu, options) {
       redraw = false;
     if (redraw) options.redraw();
   };
+  // better to only call blur when we know it's focused. Could reuse this logic in this file
+  options.ensure_blur = () => {
```

Comment:
We can have this have the same name to avoid users of the API knowing about internals:

```suggestion
  options.blur = () => {
```